lib/probes/http.js

```diff
@@ -149,9 +179,13 @@ function patchClient (module, protocol) {
   // wrap request for http and https
   shimmer.wrap(module, 'request', wrapper)
 
-  // only wrap get for http if at least node 8
-  if (protocol === 'http' && semver.satisfies(process.version, '>=8')) {
-    shimmer.wrap(module, 'get', wrapper)
+  // wrap get if at least node 8. there are a few versions
```

Comment:
This comment is a bit hard to understand. I don't see https for Node 8 being wrapped.
I would add what it is in the PR comment "Node 9.9.0+ https.get() no longer calls exports.request()" (... so we need to  wrap get for http and https )

Reply:
funny, i was looking at this comment and thinking the same thing. i didn't update it when i made the code changes. the issue is that back in 8.0.0 (maybe 7.x but we don't support non-LTS versions) node changed the way `get()` worked for http but not for https. then they changed https in 9.9.0. my original fix was just to change it for both based on node > 8.0.0 but that broke things. the comment is leftover from the first abortive fix.

thanks!